6-pubsub/composer.json

```diff
@@ -7,10 +7,12 @@
   "require": {
     "php": "~5.5.0|~5.6.0|~7.0.0",
     "google/apiclient": "~2.0@dev",
```

Comment:
So, `google/cloud` only supports `0.7` of `google/auth`, and `google/apiclient` supports `0.8`. However, the master branches of *both* support `0.9`, so I think it's best to wait until the next versions are tagged to change this.